packages/common/utils/src/groupBy.spec.ts

```diff
@@ -25,7 +25,25 @@ describe('groupBy 함수는', () => {
     });
   });
 
-  it('빈 Array를 넣었을 땐 빈 object를 return한다.', () => {
+  it('groups items distinguishing between uppercase and lowercase', () => {
+    const caseSensitiveInput = [
+      { name: 'apple', value: 1 },
+      { name: 'apple', value: 2 },
+      { name: 'Apple', value: 3 },
+      { name: 'banana', value: 4 },
+    ];
+
+    expect(groupBy(caseSensitiveInput, item => item.name)).toEqual({
+      apple: [
+        { name: 'apple', value: 1 },
+        { name: 'apple', value: 2 },
+      ],
+      Apple: [{ name: 'Apple', value: 3 }],
+      banana: [{ name: 'banana', value: 4 }],
+    });
+  });
+
+  it('returns an empty object when given an empty array', () => {
```

Comment:
same reason with above comment 😄

```suggestion
  it('should return an empty object when given an empty array', () => {
```

Reply:
I modified it as you told me. thanks :)

[chore: correct the test descriptions grammatically.](https://github.com/toss/slash/pull/360/commits/b6d7a8ce28e88d3194eef881977d3942911b3945)